bundlewrap/utils/text.py

```diff
@@ -56,6 +56,11 @@ def green(text):
     return "\033[32m{}\033[0m".format(text)
 
 
+@ansi_wrapper
+def grey(text):
+    return "\033[90m{}\033[0m".format(text)
```

Comment:
That’s problematic as it doesn’t produce meaningful output when using a bright terminal color scheme. Better go with blue.

![bright terminal](https://gist.githubusercontent.com/vain/dd8401767736bd5c4a822bbfd404fd44/raw/25635d05a26e369161620847accb48ab9d5bd127/gimp--2020-05-20--08-09-06--bright.png)